src/meta/src/hummock/hummock_manager.rs

```diff
@@ -188,7 +189,14 @@ where
         let compaction_statuses = CompactStatus::list(self.env.meta_store())
             .await?
             .into_iter()
-            .map(|cg| (cg.compaction_group_id(), cg))
+            .map(|mut cg| {
```

Comment:
Will make HashMappingSelector info part of prost::CompactStatus better or not?
I'm in favor of letting protobuf tells which LevelSelectors to use. Please refer to [this TODO](https://github.com/singularity-data/risingwave/blob/26a2e2f09e14fb931988888556e753134f67565a/src/meta/src/hummock/compaction/prost_type.rs#L69), which is error-prone.

Reply:
> Will make HashMappingSelector info part of prost::CompactStatus better or not?
> I'm in favor of letting protobuf tells which LevelSelectors to use. Please refer to [this TODO](https://github.com/singularity-data/risingwave/blob/26a2e2f09e14fb931988888556e753134f67565a/src/meta/src/hummock/compaction/prost_type.rs#L69), which is error-prone.

but we can't build HashMappingManagerRef here since we don't have context

Reply:
> this TODO

just updated comment here